Tilers/CityTiler/atlas.py

```diff
@@ -0,0 +1,135 @@
+import sys
+import numpy as np
+from PIL import Image, ImageDraw
+from py3dtiles import BoundingVolumeBox, TriangleSoup
+
+class Rectangle(object):
+
+    def __init__(self, left, top, right, bottom):
+
```

Comment:
Remove this blank line: refer to https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Was I drunk ? This WAS pep8 already